.gitmodules

```diff
@@ -1,4 +1,4 @@
 [submodule "faiss-sys/faiss"]
 	path = faiss-sys/faiss
-	url = https://github.com/Enet4/faiss.git
-	branch = c_api_head
+	url = https://github.com/aalekhpatel07/faiss.git
```

Comment:
I have just updated my fork to include the branches `v1.12.0` and `c_api/v1.12.0`, so they can be used. It should also be fine to use the main remote repository from facebookresearch, the use of my own fork was primarily to accommodate the possible need of having to introduce changes not yet merged upstream, but I think this isn't necessary at the moment.